### DISCUSSION TITLE
Water usage is always set to high after cleaning

### DISCUSSION THREAD

**Discussion by maximilian-lewald** [Hypfer/Valetudo#1798] [Q&A / Support] [answered]
When the robot finishes cleaning or is manually stopped the „Water usage“ is always set to „high“ instead of leaving it as is.
The robot used is a Roborock S7 with the Auto empty dock running Valetudo 2023.03.0.

Is there any way to prevent this?

**Comment by Hypfer** [comment, marked as answer]
On roborock robots, the "off" preset seems to behave as a one-time setting that gets reset after a task was finished.

Does the "as is" refer to the "off" position?

**Reply by maximilian-lewald** [reply]
> On roborock robots, the "off" preset seems to behave as a one-time setting that gets reset after a task was finished.

Ok nice to know. Seems counterintuitive for me..

> Does the "as is" refer to the "off" position?

Yes
